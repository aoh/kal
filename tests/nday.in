# flags: -t 0 -n 10 -r
every 2 days: repeating todo

1.1.1970
 - foo
 
2.1.1970
 + bar todo

3.1.1970
 - baz 
