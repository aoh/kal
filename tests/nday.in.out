1.1.1970, Thursday, week 1
 - foo

2.1.1970, Friday, week 1
 + bar todo
 + repeating todo

3.1.1970, Saturday, week 1
 - baz 

every 2 days: repeating todo
